src/less/pagination/ds6/pagination-base.less

```diff
@@ -0,0 +1,103 @@
+.pagination {
+    align-items: center;
```

Comment:
@seangates I realised that as I am not using flexbox in default case I can move this to `pagination--fluid` modifier